06.bowling_object/frame.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+require './shot'
+
+class Frame
+  attr_reader :first_shot, :second_shot, :third_shot
```

Comment:
```suggestion
  attr_reader :shots
```

みたいな配列にして、`shots[0], shots[1], shots[2]`のように取得すればいいのでは？と思いました。

Reply:
ありがとうございます。配列の方がやりやすかったです。